packages/jest-config/src/readConfigFileAndSetRootDir.ts

```diff
@@ -116,6 +116,7 @@ async function registerTsNode(): Promise<Service> {
     return tsNode.register({
       compilerOptions: {
         module: 'CommonJS',
+        moduleResolution: 'Node10',
```

Comment:
does `Node10` work in all the versions of TS we support? 5.0.x is the lowest we test against.

Reply:
I cannot find any information in TS release notes.

So I installed `typescript@5.0` (which resolves to `5.0.4`) and ran `tsc` on one of my projects that specify `Node10` -- it worked fine.

Reply:
perfect 👍